airflow/models.py

```diff
@@ -2630,6 +2630,65 @@ def xcom_pull(
             include_prior_dates=include_prior_dates)
 
 
+class ExecutorQueue(Base):
+    __tablename__ = "executor_queue"
+
+    key = Column(String(ID_LEN), primary_key=True)
+    id = Column(String(ID_LEN), nullable=False)
+    updated = Column(DateTime, nullable=False)
+
+
+class ExecutorQueueManager(dict):
+    def __init__(self, *args, **kwargs):
+        super(ExecutorQueueManager, self).__init__(*args, **kwargs)
+        self.session = settings.Session()
```

Comment:
This pattern seems to be different than our current pattern. Any particular reason why?

Reply:
What part?

Reply:
Session is usually the last arg to functions.